lib/v2.recommended.yaml

```diff
@@ -84,17 +83,14 @@ analyzer:
     avoid_returning_null_for_void: warning
     avoid_setters_without_getters: warning
     avoid_slow_async_io: warning
+    avoid_type_to_string: warning
```

Comment:
I hadn't seen this one, but I think it makes sense. 

the .toString in these situations is just extra information to parse.

Reply:
More than that, using `runtimeType` as a String doesn't work as expected in minified code, so any check relying on that behavior likely won't work. For debugging purposes, we should still be able to just log the object itself rather than its type.